# Turkish translation for linuxmint
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2016-04-22 13:59+0100\n"
"PO-Revision-Date: 2016-06-22 14:01+0000\n"
"Last-Translator: kelebek333 <Unknown>\n"
"Language-Team: Turkish <tr@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2016-06-25 09:26+0000\n"
"X-Generator: Launchpad (build 18111)\n"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:111
#: usr/lib/linuxmint/mintbackup/mintbackup.py:252
#: usr/lib/linuxmint/mintbackup/mintbackup.py:449
#: usr/lib/linuxmint/mintbackup/mintbackup.py:467
#: usr/lib/linuxmint/mintbackup/mintbackup.py:533
#: usr/lib/linuxmint/mintbackup/mintbackup.py:536
#: usr/lib/linuxmint/mintbackup/mintbackup.py:588
#: usr/lib/linuxmint/mintbackup/mintbackup.py:591
#: usr/lib/linuxmint/mintbackup/mintbackup.py:611
#: usr/lib/linuxmint/mintbackup/mintbackup.py:634
#: usr/lib/linuxmint/mintbackup/mintbackup.py:641
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1498
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1596
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1604
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1656 generate_desktop_files.py:26
#: generate_desktop_files.py:41 generate_desktop_files.py:56
msgid "Backup Tool"
msgstr "Yedekleme Aracı"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:169
msgid "Preserve structure"
msgstr "Yapıyı koru"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:171
msgid ".tar file"
msgstr ".tar dosyası"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:172
msgid ".tar.bz2 file"
msgstr ".tar.bz2 dosyası"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:173
msgid ".tar.gz file"
msgstr ".tar.gz dosyası"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:179
msgid "Never"
msgstr "Hiçbir zaman"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:180
msgid "Size mismatch"
msgstr "Boyut uyumsuzluğu"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:181
msgid "Modification time mismatch"
msgstr "Değişiklik zamanı uyuşmazlığı"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:182
msgid "Checksum mismatch"
msgstr "Dosya denetleme hatası"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:183
msgid "Always"
msgstr "Her Zaman"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:205
msgid "Excluded paths"
msgstr "Dahil edilmeyen yollar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:215
msgid "Type"
msgstr "Tür"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:219
msgid "Detail"
msgstr "Ayrıntı"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:225
#: usr/lib/linuxmint/mintbackup/mintbackup.py:234
msgid "Path"
msgstr "Yol"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:228
#: usr/lib/linuxmint/mintbackup/mintbackup.py:237
msgid "Error"
msgstr "Hata"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:269
msgid "Store?"
msgstr "Depola?"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:272
#: usr/lib/linuxmint/mintbackup/mintbackup.py:285
msgid "Name"
msgstr "İsim"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:282
msgid "Install"
msgstr "Kur"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:290
msgid "Backup or Restore"
msgstr "Yedekle veya Geri Yükle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:291
msgid "Choose from the following options"
msgstr "Aşağıdaki seçeneklerden seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:293
#: usr/lib/linuxmint/mintbackup/mintbackup.py:309
#: usr/lib/linuxmint/mintbackup/mintbackup.py:323
#: usr/lib/linuxmint/mintbackup/mintbackup.py:330
#: usr/lib/linuxmint/mintbackup/mintbackup.py:334
#: usr/lib/linuxmint/mintbackup/mintbackup.py:339
msgid "Backup files"
msgstr "Yedek dosyaları"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:294
#: usr/lib/linuxmint/mintbackup/mintbackup.py:343
#: usr/lib/linuxmint/mintbackup/mintbackup.py:353
#: usr/lib/linuxmint/mintbackup/mintbackup.py:359
#: usr/lib/linuxmint/mintbackup/mintbackup.py:364
msgid "Restore files"
msgstr "Dosyaları geri yükle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:295
#: usr/lib/linuxmint/mintbackup/mintbackup.py:368
#: usr/lib/linuxmint/mintbackup/mintbackup.py:373
#: usr/lib/linuxmint/mintbackup/mintbackup.py:379
#: usr/lib/linuxmint/mintbackup/mintbackup.py:384
msgid "Backup software selection"
msgstr "Yazılım seçimini yedekle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:296
#: usr/lib/linuxmint/mintbackup/mintbackup.py:388
#: usr/lib/linuxmint/mintbackup/mintbackup.py:393
#: usr/lib/linuxmint/mintbackup/mintbackup.py:400
msgid "Restore software selection"
msgstr "Yazılım seçimini geri yükle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:298
msgid "Make a backup of your files"
msgstr "Dosyalarınızın yedeğini alın"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:299
msgid "Save the list of installed applications"
msgstr "Kurulu uygulamaların listesini kaydedin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:300
msgid "Restore a previous backup"
msgstr "Bir önceki yedeği geri yükle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:301
msgid "Restore previously installed applications"
msgstr "Önceden kurulmuş uygulamaları geri yükle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:310
msgid "Please select a source and a destination for your backup"
msgstr "Yedeğiniz için lütfen bir kaynak ve hedef seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:311
#: usr/lib/linuxmint/mintbackup/mintbackup.py:347
#: usr/lib/linuxmint/mintbackup/mintbackup.py:355
msgid "Source:"
msgstr "Kaynak:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:312
#: usr/lib/linuxmint/mintbackup/mintbackup.py:348
msgid "Destination:"
msgstr "Hedef:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:313
#: usr/lib/linuxmint/mintbackup/mintbackup.py:349
msgid "Advanced options"
msgstr "Gelişmiş seçenekler"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:314
#: usr/lib/linuxmint/mintbackup/mintbackup.py:356
msgid "Description:"
msgstr "Tanım:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:315
msgid "Output:"
msgstr "Çıktı:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:316
#: usr/lib/linuxmint/mintbackup/mintbackup.py:350
msgid "Overwrite:"
msgstr "Üzerine Yaz:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:317
msgid "Confirm integrity"
msgstr "Bütünlüğü onayla"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:318
msgid "Follow symlinks"
msgstr "Sembolik bağları takip et"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:319
msgid "Preserve permissions"
msgstr "İzinleri koru"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:320
msgid "Preserve timestamps"
msgstr "Zaman damgalarını koru"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:324
msgid "Please select any files or directories you want to exclude"
msgstr "Lütfen hariç tutmak istediğiniz dosya veya dizinleri seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:325
msgid "Exclude files"
msgstr "Dosyaları dahil etme"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:326
msgid "Exclude directories"
msgstr "Dizinleri hariç tut"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:327
msgid "Remove"
msgstr "Kaldır"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:331
msgid "Please review the information below before starting the backup"
msgstr "Yedeklemeyi başlatmadan önce lütfen aşağıdaki bilgiyi gözden geçirin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:335
msgid "Please wait while your files are being backed up"
msgstr "Dosyalarınızın yedeği alınırken lütfen bekleyin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:336
#: usr/lib/linuxmint/mintbackup/mintbackup.py:381
msgid "Backing up:"
msgstr "Yedekleniyor:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:340
#: usr/lib/linuxmint/mintbackup/mintbackup.py:385
msgid "The backup is now finished"
msgstr "Yedekleme şimdi tamamlandı"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:344
msgid ""
"Please choose the type of backup to restore, its location and a destination"
msgstr "Lütfen geri yüklenecek yedeğin türünü, konum ve hedefini seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:345
msgid "Archive"
msgstr "Arşiv"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:346
msgid "Directory"
msgstr "Dizin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:354
msgid "Please review the information below"
msgstr "Lütfen aşağıdaki bilgiyi gözden geçirin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:360
msgid "Please wait while your files are being restored"
msgstr "Lütfen dosyalarınız geri yüklenirken bekleyin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:361
msgid "Restoring:"
msgstr "Geri yükleniyor:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:365
msgid "The restoration of the files is now finished"
msgstr "Dosyaların geri yüklemesi şimdi tamamlandı"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:369
msgid "Please choose a destination"
msgstr "Lütfen bir hedef seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:370
#: usr/lib/linuxmint/mintbackup/mintbackup.py:552
msgid "Destination"
msgstr "Hedef"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:374
msgid "The list below shows the packages you added to Linux Mint"
msgstr "Aşağıdaki liste Linux Mint'e eklemiş olduğunuz paketleri gösterir"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:375
#: usr/lib/linuxmint/mintbackup/mintbackup.py:395
msgid "Select all"
msgstr "Tümünü seç"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:376
#: usr/lib/linuxmint/mintbackup/mintbackup.py:396
msgid "Deselect all"
msgstr "Tüm seçilenleri kaldır"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:380
msgid "Please wait while your software selection is being backed up"
msgstr "Yazılım seçiminiz yedeklenirken lütfen bekleyin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:389
msgid "Please select a saved software selection"
msgstr "Lütfen kaydedilmiş bir yazılım seçimini seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:390
msgid "Software selection:"
msgstr "Yazılım seçimi:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:394
msgid "Select the packages you want to install"
msgstr "Kurmak istediğiniz paketleri seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:397
msgid "Refresh"
msgstr "Yenile"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:401
msgid "The restoration is now finished"
msgstr "Geri yükleme şimdi tamamlandı"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:402
msgid "Your package selection was restored successfully"
msgstr "Paket seçiminiz başarıyla geri yüklendi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:458
#: usr/lib/linuxmint/mintbackup/mintbackup.py:476
msgid "Invalid path"
msgstr "Geçersiz yol"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:458
#: usr/lib/linuxmint/mintbackup/mintbackup.py:476
#, python-format
msgid "%s is not located within your source directory. Not added."
msgstr "%s, kaynak dizininiz içinde yer almıyor. Eklenmedi."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:533
msgid "Please choose directories for the source and the destination"
msgstr "Lütfen kaynak hedef dizinleri için seçim yapınız"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:536
msgid ""
"Please choose different directories for the source and the destination"
msgstr "Kaynak ve hedef için lütfen farklı dizinler seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:551
msgid "Source"
msgstr "Kaynak"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:554
msgid "Description"
msgstr "Açıklama"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:558
msgid "Compression"
msgstr "Sıkıştırma"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:562
msgid "Overwrite destination files"
msgstr "Hedef dosyaların üzerine yaz"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:565
msgid "Exclude"
msgstr "Hariç Tut"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:588
msgid "Please choose a file to restore from"
msgstr "Lütfen geri yüklemede kullanılacak bir dosya seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:591
#: usr/lib/linuxmint/mintbackup/mintbackup.py:611
msgid "Please choose a destination directory"
msgstr "Lütfen bir hedef dizin seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:634
msgid "No packages need to be installed at this time"
msgstr "Şu anda herhangi bir paket kurmaya gerek yok"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:641
msgid "Please select one or more packages to install"
msgstr "Lütfen kurmak için bir veya daha fazla paket seçin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:709
#: usr/lib/linuxmint/mintbackup/mintbackup.py:710
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1204
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1206
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1513
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1514
msgid "Calculating..."
msgstr "Hesaplanıyor..."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:759
#: usr/lib/linuxmint/mintbackup/mintbackup.py:799
msgid "Skipping broken link"
msgstr "Kırık bağlantı atlanıyor"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:760
msgid "Broken link"
msgstr "Kırık bağlantı"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:763
#: usr/lib/linuxmint/mintbackup/mintbackup.py:803
msgid "Skipping link"
msgstr "Bağlantı atlanıyor"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:827
#: usr/lib/linuxmint/mintbackup/mintbackup.py:836
#: usr/lib/linuxmint/mintbackup/mintbackup.py:845
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1265
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1276
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1289
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1354
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1363
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1373
msgid "Skipping identical file"
msgstr "Tekrarlanan dosya atlanıyor"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:867
#, python-format
msgid ""
"Warning: Some files were not saved, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Uyarı: Bazı dosyalar kaydedilmedi, kopyalanan: %(total)d dosyadan "
"%(current_file)d tanesi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:871
msgid "An error occurred during the backup"
msgstr "Yedekleme sırasında bir hata oluştu"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:881
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1559
msgid "The backup was aborted"
msgstr "Yedekleme iptal edildi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:889
msgid "The backup completed successfully"
msgstr "Yedekleme başarıyla tamamlandı"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:980
msgid "Checksum Mismatch:"
msgstr "Sağlama Uyumsuzluğu:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:981
msgid "Checksum Mismatch"
msgstr "Sağlama Uyumsuzluğu"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1033
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1035
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1064
msgid "Calculating checksum"
msgstr "Sağlama yapılıyor"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1110
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1134
msgid "(Not created with the Backup Tool)"
msgstr "(Yedekleme Aracı ile oluşturulmamış)"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1397
#, python-format
msgid ""
"Warning: Some files were not restored, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Uyarı: Bazı dosyalar onarılamadı, kopyalanan: %(total)d dosyadan "
"%(current_file)d tanesi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1400
msgid "An error occurred during the restoration"
msgstr "Geri yükleme sırasında bir hata oluştu"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1411
msgid "The restoration was aborted"
msgstr "Geri yükleme iptal edildi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1419
msgid "The restoration completed successfully"
msgstr "Geri yükleme başarıyla tamamlandı"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1550
msgid "An error occurred during the backup:"
msgstr "Yedekleme sırasında bir hata oluştu :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1567
msgid "Your software selection was backed up successfully"
msgstr "Yazılım seçiminiz başarıyla yedeklendi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1596
msgid "The specified file is not a valid software selection"
msgstr "Belirtilen dosya geçerli bir yazılım seçimi değil"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1604
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1656
msgid "An error occurred while accessing the file"
msgstr "Dosyaya erişirken bir hata meydana geldi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1649
msgid "Could not locate the package"
msgstr "Paket konumu tespit edilemedi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1681
msgid "Please wait, this can take some time"
msgstr "Lütfen bekleyin, bu işlem biraz zaman alabilir"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1683
msgid "The installation is complete"
msgstr "Kurulum tamamlandı"

#: generate_desktop_files.py:26 generate_desktop_files.py:41
#: generate_desktop_files.py:56
msgid "Make a backup of your home directory"
msgstr "Ev dizininin bir yedeğini alın"

#~ msgid "Backup successful"
#~ msgstr "Yedekleme başarılı"

#~ msgid "Your home directory was successfully backed-up into"
#~ msgstr "......... içerisine ev dizininiz başarıyla yedeklenmiştir."

#~ msgid "Backup failed"
#~ msgstr "Yedekleme başarısız"

#~ msgid "Cleaning up..."
#~ msgstr "Temizleniyor..."

#~ msgid "Your backup was successfully restored"
#~ msgstr "Yedekleme başarıyla geri yüklendi"

#~ msgid "An error occurred while restoring the backup archive:"
#~ msgstr "Yedekleme arşivi geri yüklenirken bir hata oluştu :"

#~ msgid "Opening the backup archive..."
#~ msgstr "Yedekleme arşivi açılıyor..."

#~ msgid "Read error"
#~ msgstr "Okuma hatası"

#~ msgid "An error occurred while opening the backup:"
#~ msgstr "Yedekleme açılırken bir hata oluştu:"

#~ msgid "Hidden paths"
#~ msgstr "Gizli yollar"

#~ msgid "Exclude folders"
#~ msgstr "Klasörleri dahil etme"

#~ msgid "Backup"
#~ msgstr "Yedekle"

#~ msgid "Excluded Files and Directories"
#~ msgstr "Dahil edilmeyen Dosya ve Klasörler"

#~ msgid "Included"
#~ msgstr "Dahil edilen"

#~ msgid "Included hidden directories"
#~ msgstr "Dahil edilen gizli dizinler"

#~ msgid "Restore your personal data from this backup"
#~ msgstr "Kişisel bilgilerinizi bu yedekten geri yükleyin"

#~ msgid "Overwrite existing files"
#~ msgstr "Varolan dosyaların üzerine yaz"

#~ msgid "Restore"
#~ msgstr "Geri Yükle"

#~ msgid "View content"
#~ msgstr "İçeriği göster"

#~ msgid "Archiving your home directory..."
#~ msgstr "Ev dizini arşivleniyor..."

#~ msgid "Extracting the content of the backup into your home directory..."
#~ msgstr "Yedekleme dosyasının içeriği ev dizinine çıkarılıyor..."

#~ msgid "Copying the backup file into your home directory..."
#~ msgstr "Yedekleme dosyası ev dizinine kopyalanıyor..."

#~ msgid "Restoration successful"
#~ msgstr "Geri Yükleme başarılı"

#~ msgid "Restoration failed"
#~ msgstr "Geri Yükleme başarısız"

#~ msgid "<big><b>Load data into your home directory</b></big>"
#~ msgstr "<big><b>Ev dizinine veri yükle</b></big>"

#~ msgid "is not located within your home directory. Not added."
#~ msgstr "ev dizininde bulunmuyor. Eklenmemiş."
